### ISSUE TITLE
Documented 2nd arg (e.g. :hex) for hash functions doesn't seem to exist

### ISSUE THREAD

**Issue by grempe** [cryptosphere/rbnacl#114]
The wiki documents and code imply a second arg can be passed to hash functions to allow the return of different string/data types from those functions.  This does not appear to exist in the code or the tests.

https://github.com/cryptosphere/rbnacl/wiki/Hash-Functions

````
# compute the SHA-512 digest of the message and hex-encode it.
RbNaCl::Hash.sha512(message, :hex)
#=> "ab67..." # a string of hexidecimal bytes.
````

Specs & Code References:
https://github.com/cryptosphere/rbnacl/blob/master/spec/rbnacl/hash_spec.rb
https://github.com/cryptosphere/rbnacl/blob/master/lib/rbnacl/hash.rb#L24
https://github.com/cryptosphere/rbnacl/blob/master/lib/rbnacl/hash.rb#L40
https://github.com/cryptosphere/rbnacl/blob/master/lib/rbnacl/hash.rb#L58

Am I missing something?

Cheers,

Glenn

**Comment by grempe** [comment]
@namelessjon Not sure why you closed this.  The issue (referencing a non-existent second arg) is still present in the inline docs and line numbers I linked to above. I didn't think this minor doc issue needed a pull-request.  Maybe I was wrong?

**Comment by tarcieri** [comment]
Fixed in 4d7d67d
